dmscripts/generate_user_email_list_applications.py

```diff
@@ -0,0 +1,110 @@
+import csv
```

Comment:
Better to look here - for proper Python 2 / Python 3 compatibility I think you need to do conditional importing: https://github.com/alphagov/digitalmarketplace-scripts/blob/master/dmscripts/generate_framework_agreement_data.py#L7